pkg/unittest/test_suite.go

```diff
@@ -37,6 +37,9 @@ func ParseTestSuiteFile(suiteFilePath, chartRoute string, strict bool) (*TestSui
 		}
 	}
 
+	// Append the valuesfiles from command to the testsuites.
```

Comment:
I see that you are appending here the values from the parameter in cli next to whatever is specified in the test suite.

How is the merge process for same value fields? What is the priority?

Reply:
The merge process is handled in the test_job.go (getUserValues function)

The order is first the valuesfiles (command, suite, test), followed by the set values.